plugins/users_api.py

```
@@ -30,9 +24,7 @@ async def get_short_link(user, link):
     if data["status"] == "success" or rget.status_code == 200:
```

Ruff (0.8.2)

Undefined name `rget`

(F821)